docs/yarn.md

```diff
@@ -176,3 +178,4 @@ YARN_ENABLE_IMMUTABLE_CACHE=false
 [readme-yarn]: ../README.md#yarn
 [usage-prefetch]: usage.md#pre-fetch-dependencies
 [usage-genenv]: usage.md#generate-environment-variables
+[yarn-classic]: yarn_classic.md
```

Comment:
Technically, this link is not used. You can drop it or use above this `[yarn_classic.md][yarn-classic]`.